CHANGELOG.md

```diff
@@ -62,6 +62,7 @@ For semantic convention package changes, see the [semconv CHANGELOG](packages/se
     * propagator-aws-xray
     * sdk-metrics
 * deps(sdk-metrics): remove unused lodash.merge dependency [#4905](https://github.com/open-telemetry/opentelemetry-js/pull/4905) @pichlermarc
+* refactor(sdk-trace-base): replace `SpanAttributes` with `Attributes` [#5009](https://github.com/open-telemetry/opentelemetry-js/pull/5009) @david-luna
```

Comment:
ah, I think we have a `CHANGELOG_NEXT` for things like this :slightly_smiling_face:

Reply:
this would also be breaking :)

Reply:
done!